mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -718,6 +718,7 @@ def Vector_ExtractOp :
   let results = (outs AnyType:$result);
 
   let builders = [
+    OpBuilder<(ins "Value":$source)>,
```

Comment:
IIUC, this builder is for extracting from rank-0 vector? Hence no indices? It's worth adding a comment.

Note, the "source" vector is basically the "indexed" argument for `vector.extract`:
* https://github.com/llvm/llvm-project/pull/130141

So, if that taxonomy makes sense, you can write:
```mlir
// A dedicated builder for when the "indexed" operand is rank-0 and hence no indices are required.
```

Identical comment for `vector.insert` (over there it's the "destination" operand that's "indexed")

Reply:
Added a similar comment clarifying what this builder is for.